ndsl/stencils/testing/serialbox_to_netcdf.py

```diff
@@ -96,8 +113,36 @@ def main(data_path: str, output_path: str, data_name: Optional[str] = None):
                     rank_data[name].append(
                         read_serialized_data(serializer, savepoint, name)
                     )
+                nblocks = len(rank_data[name])
+                if merge_blocks and len(rank_data[name]) > 1:
+                    full_data = np.array(rank_data[name])
+                    if len(full_data.shape) > 1:
+                        if (nx * ny == full_data.shape[0] * full_data.shape[1]):
+                            # If we have an (i, x) array from each block reshape it
+                            new_shape = (nx, ny) + full_data.shape[2:]
+                            full_data = full_data.reshape(new_shape)
+                        else:
+                            # We have one array for all blocks
+                            # could be a k-array or something else, so we take one copy
+                            # TODO: is there a decent check for this?
+                            full_data = full_data[0]
+                        #else:
```

Comment:
Remove the code below.
Add a print above (maybe only printed once per execution) to feedback that we are doing dubious copies